### ISSUE TITLE
Backport: Rancher-monitoring: query issue on daemonset 

### ISSUE THREAD

**Issue by alena1108** [rancher/rancher#20242]
Backport https://github.com/rancher/rancher/issues/20162

**Comment by alena1108** [comment]
The fix is in system-charts dev branch, and can be tested with v2.2.4-rc1

**Comment by davidnuzik** [comment]
https://github.com/rancher/system-charts/pull/43

**Comment by davidnuzik** [comment]
Version: v2.2.4-rc1

The PR https://github.com/rancher/system-charts/pull/43 contains a fix and here is the overview:

>Problem: There are two kubelet scraping targets on Prometheus, one is scraping /metrics, another one is scraping /metrics/cadvisor. The metrics from /metrics endpoint are not including container_name. So the container_* expression will double the actual mount without
conatiner_name!=""

>Fix: Add `container_name!=""` into the expression

This should fix the inaccurate Grafana dashboards that were in some cases (when `container_name!=""` was not there) showing 2x the values.

I tested again and now I observe data that matches what we see in Rancher charts. Grafana is no longer showing 2x the values.

I tested by creating a CPU load on an ubuntu workload. I utilized the `stress` package to put 50% of cpu cores under load then 100% cpu cores under load. This way we have consistent cpu utilization more or less. I ensured that for each test the Rancher wheels and charts matched what we see in Grafana -- down to the Deployment level this appears accurate now (check Cluster, Node, Pod, Deployment [memory], etc and compare each to Rancher UI). I did spot checks as well for memory usage, etc which also appears accurate now.
